validator/src/validator.rs

```diff
@@ -105,22 +87,15 @@ where
     table: ValidatorTable,
     env: MdbxDatabase,
 
-    validator_address: Arc<RwLock<Address>>,
-    signing_key: Arc<RwLock<SchnorrKeyPair>>,
-    voting_keys: Arc<RwLock<VotingKeys>>,
-    fee_key: Arc<RwLock<SchnorrKeyPair>>,
+    state: Arc<RwLock<ValidatorState>>,
+    inactivity_state: Option<InactivityState>,
 
     proposal_receiver: ProposalReceiver<TValidatorNetwork>,
 
     consensus_event_rx: BroadcastStream<ConsensusEvent>,
     network_event_rx: SubscribeEvents<<TValidatorNetwork::NetworkType as Network>::PeerId>,
     fork_event_rx: BroadcastStream<ForkEvent>,
 
-    slot_band: Arc<RwLock<Option<u16>>>,
-    consensus_state: Arc<RwLock<ConsensusState>>,
-    validator_state: Option<InactivityState>,
-    automatic_reactivate: Arc<AtomicBool>,
-
     macro_producer: Option<ProduceMacroBlock<TValidatorNetwork>>,
     macro_state: Arc<RwLock<Option<MacroState>>>,
```

Comment:
This could be added to the `ValidatorState`. Additionally that would open up a Rpc call to see the macro state which could be a very useful debugging tool.